src/styles/main.scss

```diff
@@ -1,3 +1,248 @@
+  margin-right: 128px;
+
+  @media (max-width:1024px) {
+    margin-left: 40px;
+    margin-right: 40px;
+  }
+
+  @media (min-width: 1201px) {
+    margin-right: 160px;
+    margin-left: 160px;
+  }
+}
+
+.card {
+  width: 198px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: 46px;
+
+  @media (min-width: 1201px) {
+    margin-right: 20px;
+    margin-left: 20px;
+  }
+
+  &-picture {
+    padding: 32px 19px 36px;
+    width: 160px;
+    height: 134px;
+    transition: 400ms;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+  }
+
+  &-product__description {
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0 16px;
+    color: #060b35;
+  }
+
+  &-product__code {
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 10px;
+    line-height: 14px;
+    margin-left: 16px;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    color: #616070;
+  }
+
+  &-rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 14px;
+    margin-left: 16px;
+    margin-bottom: 24px;
+
+    &__stars {
+      display: flex;
+
+      & > :not(:last-child) {
+        margin-right: 4px;
+      }
+    }
+
+    &__feedback {
+      display: flex;
+      font-family: Roboto, sans-serif;
+      font-style: normal;
+      font-weight: normal;
+      font-size: 10px;
+      margin-right: 16px;
+      color: #060b35;
+    }
+  }
+
+  &-price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &__word {
+      font-family: Roboto, sans-serif;
+      font-style: normal;
+      font-weight: normal;
+      font-size: 12px;
+      line-height: 18px;
+      color: #616070;
+      margin-left: 16px;
+    }
+
+    &__number {
+      font-family: Roboto, sans-serif;
+      font-style: normal;
+      font-weight: bold;
+      font-size: 16px;
+      line-height: 18px;
+      margin-right: 16px;
+      display: flex;
+      color: #060b35;
+    }
+  }
+
+  &-buy {
+    font-family: Roboto, sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 16px;
+    width: 166px;
+    height: 40px;
+    background: #00acdc;
+    color: #fff;
+    border-radius: 5px;
+
+    transition-duration: 400ms;
+
+    &:hover {
+      box-sizing: border-box;
+      background: #fff;
+      color: #00acdc;
+      border: 1px solid #00acdc;
+    }
+  }
+}
+
+.str {
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.stars-4 :nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
You should split your main.scss to separate files, with a different blocks (for examle: `header.scss` `card.scss` etc..)